### PR TITLE
Modified Readme intro

### PR DESCRIPTION
**Pull request by ShyamW**
### Description

Replace this with the PR description (mention the changes you have made, why
you have made them, provide some background and any references to the provider
documentation if needed, etc.).

For more information on contributing, please see [Contributing](http://libcloud.readthedocs.org/en/latest/development.html#contributing)
section of our documentation.
### Status

Replace this: describe the PR status. Examples:
- work in progress
- done, ready for review
### Checklist (tick everything that applies)
- [ ] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [ ] Documentation
- [ ] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [ ] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

### CODE REVIEW

**Comment by ShyamW** [comment]
Simplified Intro

File: README.rst [hunk 1]
```diff
@@ -31,9 +31,8 @@ Apache Libcloud - a unified interface into the cloud
 .. image:: https://coveralls.io/repos/github/apache/libcloud/badge.svg?branch=trunk
         :target: https://coveralls.io/github/apache/libcloud?branch=trunk
 
-Apache Libcloud is a Python library which hides differences between different
-cloud provider APIs and allows you to manage different cloud resources
-through a unified and easy to use API.
+Apache Libcloud is a Python library which abstracts many different
```
Comment (lakshmi-kannan) [inline README.rst]: s/which/that

**Review by ShyamW — COMMENTED** [review]
Better?

File: README.rst (hunk 1 above)
Comment (ShyamW) [reply README.rst]: Better?

File: README.rst [hunk 2]
```diff
@@ -31,9 +31,8 @@ Apache Libcloud - a unified interface into the cloud
 .. image:: https://coveralls.io/repos/github/apache/libcloud/badge.svg?branch=trunk
         :target: https://coveralls.io/github/apache/libcloud?branch=trunk
 
-Apache Libcloud is a Python library which hides differences between different
-cloud provider APIs and allows you to manage different cloud resources
-through a unified and easy to use API.
+Apache Libcloud is a Python library that wraps many different
+cloud provider APIs including: ...
```
Comment (tonybaloney) [inline README.rst:35]: There's no follow on to this sentence, it doesn't make any sense now. I'm closing this PR.

**Pull request closed without merging**